Add some missing inlines on psychic class feats

failwiz opened the pull request:
Affected feats: **Brain Drain**, **Shatter Space**, **Target of Psychic Ire**
Also changed localization strings for **Shatter Space** and **Target of Psychic Ire** amps.

failwiz committed:
Add some missing inlines on psychic class feats

ammalagonc committed:
Correct action cost of Unholy Army (#20934)

stwlam committed:
Move early validation of `CreatureSizeRuleElement#value` to schema definition (#20936)

stwlam committed:
Fix reference to core icon in _coinage.scss (#20937)

stwlam committed:
Use ancestry reach value when setting PC reach from ancestry (#20938)

simonward commented on packs/feats/class/psychic/shatter-space.json:
```diff
@@ -12,7 +12,7 @@
         "category": "class",
         "description": {
-            "value": "<p>The sheer power of your magic fractures reality. Use this amp in place of a psi cantrip's normal amp entry. The cantrip must be one that takes 2 or more actions to cast.</p>\n<hr />\n<p><strong>Amp</strong> You or one target of the spell are surrounded by a spiral of torn space. This is an aura, and you choose the radius of the emanation, up to 20 feet. A creature that st ...
+            "value": "<p>The sheer power of your magic fractures reality. Use this amp in place of a psi cantrip's normal amp entry. The cantrip must be one that takes 2 or more actions to cast.</p>\n<hr />\n<p><strong>Amp</strong> You or one target of the spell are surrounded by a spiral of torn space. This is an aura, and you choose the radius of the emanation, up to 20 feet. A creature that st ...
```
```suggestion
            "value": "<p>The sheer power of your magic fractures reality. Use this amp in place of a psi cantrip's normal amp entry. The cantrip must be one that takes 2 or more actions to cast.</p>\n<hr />\n<p><strong>Amp</strong> You or one target of the spell are surrounded by a spiral of torn space. This is an aura, and you choose the radius of the emanation, up to 20 feet. A creature that starts its turn in the aura or enters the aura during its turn takes 1d6 slashing damage for every level of the amped psi cantrip, with a @Check[reflex|against:occult|basic|options:area-effect,inflicts:prone] save against your spell DC. If the creature critically fails, the torn space throws it off balance and it falls @UUID[Compendium.pf2e.conditionitems.Item.Prone]. The aura lasts until the beginning of your next turn. You can choose whether the aura affects the creature emitting it.</p>"
```

simonward commented on static/lang/re-en.json:
```diff
@@ -5771,13 +5771,13 @@
                     "RemovePresence": "Choose one target of the spell or one creature in its area. If that enemy fails its save or the spell hits it, you become undetected by that creature, disappearing from its senses for 1 minute or until you use a hostile action. This is similar to being @UUID[Compendium.pf2e.conditionitems.Item.zJxUflt9np0q4yML]{Invisible}, but effects like see invisibility d ...
                     "ShatterMind": "You increase the area of the spell to your choice of a @Template[cone|distance:30] or @Template[cone|distance:60], and the damage dice for the spell change to d10s. Creatures that fail are @UUID[Compendium.pf2e.conditionitems.Item.e1XGnhKNSQIm5IXg]{Stupefied 1} until the start of your next turn (or @UUID[Compendium.pf2e.conditionitems.Item.e1XGnhKNSQIm5IXg]{Stu ...
-                    "ShatterSpace": "You or one target of the spell are surrounded by a spiral of torn space. This is an aura, and you choose the radius of the emanation, up to 20 feet. A creature that starts its turn in the aura or enters the aura during its turn takes @Damage[(@item.level)d6[slashing]] damage, with a basic Reflex save against your spell DC. If the creature critically fails, the ...
+                    "ShatterSpace": "You or one target of the spell are surrounded by a spiral of torn space. This is an aura, and you choose the radius of the emanation, up to 20 feet. A creature that starts its turn in the aura or enters the aura during its turn takes @Damage[(@item.level)d6[slashing]] damage, with a @Check[reflex|against:occult|basic|options:area-effect] save against your spel ...
```
```suggestion
                    "ShatterSpace": "You or one target of the spell are surrounded by a spiral of torn space. This is an aura, and you choose the radius of the emanation, up to 20 feet. A creature that starts its turn in the aura or enters the aura during its turn takes @Damage[(@item.level)d6[slashing]|options:area-damage] damage, with a @Check[reflex|against:occult|basic|options:area-effect,inflicts:prone] save against your spell DC. If the creature critically fails, the torn space throws it off balance and it falls @UUID[Compendium.pf2e.conditionitems.Item.j91X7x0XSomq8d60]{Prone}. The aura lasts until the beginning of your next turn. You can choose whether the aura affects the creature emitting it.",
```

simonward commented on packs/feats/class/psychic/target-of-psychic-ire.json:
```diff
@@ -12,7 +12,7 @@
         "category": "class",
         "description": {
-            "value": "<p>Your magic saturates your surroundings with hateful psychic energy. Use this amp in place of a psi cantrip's normal amp entry. The cantrip must be one that takes 2 or more actions to cast and targets a creature.</p>\n<hr />\n<p><strong>Amp</strong> Choose one target of the spell. Psychic phenomena turn themselves on the target-typically, objects fling themselves at the cr ...
+            "value": "<p>Your magic saturates your surroundings with hateful psychic energy. Use this amp in place of a psi cantrip's normal amp entry. The cantrip must be one that takes 2 or more actions to cast and targets a creature.</p><hr /><p><strong>Amp</strong> Choose one target of the spell. Psychic phenomena turn themselves on the target-typically, objects fling themselves at the creatu ...
```
```suggestion
            "value": "<p>Your magic saturates your surroundings with hateful psychic energy. Use this amp in place of a psi cantrip's normal amp entry. The cantrip must be one that takes 2 or more actions to cast and targets a creature.</p><hr /><p><strong>Amp</strong> Choose one target of the spell. Psychic phenomena turn themselves on the target-typically, objects fling themselves at the creature. At the beginning of each of the target's turns, it takes 1d4 bludgeoning damage for each spell rank of the psi cantrip this amp was added to, with a @Check[reflex|against:occult|basic] save. The target remains marked by psychic ire for 3 rounds. The effect persists even if you fall @UUID[Compendium.pf2e.conditionitems.Item.Unconscious] or leave the area.</p>"
```

failwiz committed:
Apply suggestions from code review

I'll remember `inflicts` next time, thanks. And no damage inlines in the feat text of amp because no one actually rolls from that, right?

Co-authored-by: Simon Ward <simonward@mac.com>

failwiz commented:
Wow, that's an embarrassing commit message! That's what I get for not actually waking up before checking mail.

SpartanCPA committed:
Add Twisting Tree automation for Onyiji (#20943)

pedrogrullada committed:
Flip multiplier of held ctrl/shift keys in Trade Dialog to match actor sheet inventory (#20945)

stwlam committed:
Enact system-split of compendium data (#20946)

ammalagonc committed:
Correct Dvorovoi's Master of the Yard's description (#20947)

ammalagonc committed:
Update Domovoi's Home Guardian to use Action Macros (#20948)

pedrogrullada commented:
There's conflicts because of the recent compendium split. You need to rebase. (You might be the person who asked on Discord, in which case you're already aware probably :P)

stwlam committed:
Add single Starfinder item to packs/sf2e (#20949)

failwiz committed:
Merge branch 'v13-dev' into psychic-feats-missing-inlines

ammalagonc committed:
Rebase

TikaelSol commented:
Sorry, caused more conflicts for you. Lots of moving parts with feats right now

failwiz commented:
Not a problem, I'll rebase when I'm at my PC (and/or when things settle down).

TikaelSol commented:
Things should be mostly settled tomorrow, last big wrecking ball coming through in a few minutes

failwiz committed:
Merge branch 'v13-dev' into psychic-feats-missing-inlines
